README.md

```diff
@@ -1,43 +1,54 @@
-# Footer
+VTEX Footer
+=====
+
+## Description
+The VTEX Footer app shows information about the store such as address, social networks and payment methods. Furthermore, it is possible to add hyperlinks for privacy policy, FAQ, benefits and attendance. This is a VTEX app that is used by Dreamstore product.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
-Footer is a canonical component that any VTEX app can import.
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-09-20     | 2018-11-21            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-21     |                       |             | 2.x
 
 
-To import it into your code:
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-```js
-import { Footer } from 'vtex.store-footer'
-```
+## Table of Contents
+- [Usage](#usage)
+- [API](#api)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
+
+## Usage
+
+This app use our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-Also, you can import as a dependency in your `manifest.json`
+To use this app you need to import in your dependencies on `manifest.json`.
 
 ```json
   dependencies: {
     "vtex.store-footer: 2.x"
   }
 ```
 
-## Usage
-#### Deprecated
-
-You can use it in your code like a React component with the jsx tag: `<Footer />`.
+Then, add `footer` block into our app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json).
 
-```jsx
-<Footer />
-```
+### Blocks API
+:construction: :construction: :construction:
 
-Or, you can add in your `pages.json`:
+This app have an interface that describes what are the rules that the blocks must implement when you want to use the footer block. 
 
 ```json
- "store/footer": {
-      "component": "vtex.dreamstore-footer/index"
- }
+  "footer": {
+    "component": "index"
+  }
 ```
+#### Configuration
+Through the Storefront you can change the behavior and interface of the footer. But, you can also make adjusts in your theme app, like Dreamstore does.
 
 | Prop name | Type | Description |
 | --- | --- | --- |
```

Comment:
Double check if the props are correct. I noticed that `moreInformationLinks` prop does not exist anymore.